progressbar/widgets.py

```diff
@@ -260,7 +260,8 @@ def __init__(
     def _calculate_eta(self, progress, data, value, elapsed):
         '''Updates the widget to show the ETA or total time when finished.'''
         if elapsed and value:
-            eta_seconds = elapsed * progress.max_value / value - elapsed
```

Comment:
In retrospect, you are right. It's the delta vs. the complete value... that was wrong :)